src/Tizen.NUI/src/public/BaseComponents/CustomView.cs

```diff
@@ -164,10 +166,37 @@ public virtual void OnStageConnection(int depth)
         /// When the parent of a set of views is disconnected to the stage, then all of the children will receive this callback, starting with the leaf views.<br />
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated since API level 8 and will be removed in API level 10. Please use OnSceneDisconnection instead!")]
         public virtual void OnStageDisconnection()
         {
         }
 
+        /// <summary>
+        /// Called after the view has been connected to the scene.<br />
+        /// When a view is connected, it will be directly or indirectly parented to the root view.<br />
+        /// The root view is provided automatically by Tizen.NUI.Window, and is always considered to be connected.<br />
+        /// When the parent of a set of views is connected to the scene, then all of the children will receive this callback.<br />
+        /// </summary>
+        /// <param name="depth">The depth in the hierarchy for the view.</param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void OnSceneConnection(int depth)
+        {
+        }
+
+        /// <summary>
+        /// Called after the view has been disconnected from the scene.<br />
+        /// If a view is disconnected, it either has no parent, or is parented to a disconnected view.<br />
+        /// When the parent of a set of views is disconnected to the scene, then all of the children will receive this callback, starting with the leaf views.<br />
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Should I remove EditorBrowsableAttribute for the new APIs I added? So both the deprecated and new APIs should be visible to public?